File: Math and Logic/Fibonacci/Fibonacci.cpp
// This program is for displaying Fibonacci series upto n numbers.
// This follows an iterative approach.

#include<iostream>
using namespace std;

void fibo(int n)
{
    int sum=0,term1=0,term2=1;
    for(int i = 2; i <= n; i++)
    {
        cout<<sum<<" ";
        sum = term1 + term2;
        term1 = term2;
        term2 = sum;
    }
    
}

int main()
{
    int n;
    cout<<"Enter the number till which Fibonacci series is to be displayed: "<<endl;
    cin>>n;

    fibo(n);
    
    return 0;
}

File: Math and Logic/Fibonacci/Fibonacci.py
# Program to display Fibonacci Series till n numbers

def fibonacci(n):
    a,b=0,1 # starts from
    while a<n:
        print(a,end=' ')
        a,b=b,a+b


if __name__ == '__main__':
    n = int(input('Enter the number of terms for fibonacci series: '))
    fibonacci(n) 